core-bundle/contao/dca/tl_user.php

```diff
@@ -619,15 +621,32 @@ public function sessionField(DataContainer $dc)
 			}
 		}
 
+		$operations = array();
+		$labels = array(
+			'purge_session' => $GLOBALS['TL_LANG']['tl_user']['sessionLabel'],
+			'purge_images' => $GLOBALS['TL_LANG']['tl_user']['htmlLabel'],
+			'purge_previews' => $GLOBALS['TL_LANG']['tl_user']['previewLabel'],
+			'purge_pages' => $GLOBALS['TL_LANG']['tl_user']['tempLabel'],
+		);
+
+		foreach ($allowedOperations as $i => $operation)
+		{
+			$operations[] = sprintf('<span><input type="checkbox" name="purge[]" id="opt_purge_%d" class="tl_checkbox" value="%s" data-action="focus->contao--scroll-offset#store"> <label for="opt_purge_%d">%s</label></span>',
+				$i,
+				$operation,
+				$i,
+				$labels[$operation]
+			);
+		}
+
+		$operations = implode('<br>', $operations);
+
 		return '
 <div class="widget">
   <fieldset class="tl_checkbox_container">
     <legend>' . $GLOBALS['TL_LANG']['tl_user']['session'][0] . '</legend>
-    <input type="checkbox" id="check_all_purge" class="tl_checkbox" onclick="Backend.toggleCheckboxGroup(this, \'ctrl_purge\')"> <label for="check_all_purge" class="check-all"><em>' . $GLOBALS['TL_LANG']['MSC']['selectAll'] . '</em></label><br>
-    <input type="checkbox" name="purge[]" id="opt_purge_0" class="tl_checkbox" value="purge_session" data-action="focus->contao--scroll-offset#store"> <label for="opt_purge_0">' . $GLOBALS['TL_LANG']['tl_user']['sessionLabel'] . '</label><br>
-    <input type="checkbox" name="purge[]" id="opt_purge_1" class="tl_checkbox" value="purge_images" data-action="focus->contao--scroll-offset#store"> <label for="opt_purge_1">' . $GLOBALS['TL_LANG']['tl_user']['htmlLabel'] . '</label><br>
-    <input type="checkbox" name="purge[]" id="opt_purge_2" class="tl_checkbox" value="purge_previews" data-action="focus->contao--scroll-offset#store"> <label for="opt_purge_2">' . $GLOBALS['TL_LANG']['tl_user']['previewLabel'] . '</label><br>
-    <input type="checkbox" name="purge[]" id="opt_purge_3" class="tl_checkbox" value="purge_pages" data-action="focus->contao--scroll-offset#store"> <label for="opt_purge_3">' . $GLOBALS['TL_LANG']['tl_user']['tempLabel'] . '</label>
+    <span><input type="checkbox" id="check_all_purge" class="tl_checkbox" onclick="Backend.toggleCheckboxGroup(this, \'ctrl_purge\')"> <label for="check_all_purge" class="check-all"><em>' . $GLOBALS['TL_LANG']['MSC']['selectAll'] . '</em></label><br>
```

Comment:
Any reason for the newly introduced `<span>` for this and the operations?
(Not that they'd do any harm, just asking)